src/lib/tabs/tab-body.ts

```diff
@@ -29,7 +29,9 @@ import {
 } from '@angular/animations';
 import {TemplatePortal, PortalHostDirective} from '@angular/cdk/portal';
 import {Directionality, Direction} from '@angular/cdk/bidi';
-
+/* tslint:disable */
```

Comment:
Can you change this to just the next-line variant with the specific lint rule mentioned?

Reply:
Done :)